src/style.css

```diff
@@ -0,0 +1,42 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+}
+
+.div_with_links {

```

Comment:
Don't use tag names in class names. class should describe meaning or purpose of the element.